content/software/ide-v1/tutorials/Linux/Linux.md

```diff
@@ -27,7 +27,17 @@ The file is compressed and you have to extract it in a suitable folder, remember
 
 Open the **arduino-1.6.x** folder just created by the extraction process and spot the **install.sh** file. Right click on it and choose **Run in Terminal** from the contextual menu. The installation process will quickly end and you should find a new icon on your desktop.
 
-If you don't find the option to run the script from the contextual menu, you have to open a Terminal window and move into the **arduino-1.6.x** folder. Type the command **assets/install.sh** and wait for the process to finish. You should find a new icon on your desktop.
+If you don't find the option to run the script from the contextual menu, you have to open a Terminal window and move into the **arduino-1.6.x** root directory and run the following command:
+
+```
+sudo sh install.sh
+```
+NOTE: 
+- For version>=1.6.5
+- Remove 'sudo' keyword if already running terminal with administrative rights. 
```

Comment:
```suggestion
***This method only works for version greater or equal than 1.6.5.  Remove `sudo` keyword if already running terminal with administrative rights.***
```

To follow our style guide! 😄 
So this will be inside in a note snippet.

After you accept this, I'll merge it :)

Reply:
I have accepted it. :smile: